integrated_channels/blackboard/migrations/0011_alter_blackboardlearnerassessmentdatatransmissionaudit_enterprise_course_enrollment_id.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 3.2.11 on 2022-01-24 15:05
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('blackboard', '0010_auto_20211221_1532'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='blackboardlearnerassessmentdatatransmissionaudit',
+            name='enterprise_course_enrollment_id',
```

Comment:
so @awais786 in essence, this PR is about changing this field to IntegerField instead of PositiveIntegerField?

Reply:
just to understand better, it's only an issue for these two tables because the table names + field constraint names combined are too long?

Reply:
I am seeing no reason to reject the PR since it looks like it avoids generating those long constraint by using a regular IntegerField instead of PositiveIntegerField, but looks also like a better longer term fix would be to use shorter model/table names

Reply:
What I don't understand is why Django generates migrations for the other models that did not change... ?

Reply:
- @binodpant On mysql8 when migration runs they generate internal check and due to big table+check name they cross the limit and crashes. 
- We faced this issue only in 2 tables and now [migrations](https://github.com/openedx/edx-platform/runs/4965788270?check_suite_focus=true) are running fine on mysql8.
- changing table is name is risky and time consuming since it has million of records. For new table we will add a check to make sure model name has normal length.
- I have squashed the migrations and generate new one.

Reply:
Removed the squashed migrations those will come in next PR after deployment.

Reply:
Got it, I think we are fine. @awais786  We can also just disable the configs temporarily from django admin, to avoid processing this table, if you feel the need for that. we only have a handful blackboard configs. Depending on timeframe, we can inform @mattdrayer and act accordingly.

Reply:
I meant Blackboard _and_ SAP configs

Reply:
My PR is ready we can do this next week @nadeemshahzad.

Reply:
@binodpant can you help me in deployment with disabling the configs temporarily from django admin ?